python/smashbox/utilities/__init__.py

```diff
@@ -657,3 +686,18 @@ def expect_does_not_exist(fn):
     """ Checks that a file does not exist, as expected
     """
     error_check(not os.path.exists(fn), "File %s exists but should not" % fn)
+
+
+def download_file(url, filename):
+    """Downloads a file in a path given (filename) if just the name of the file is given 
+       it is download locally
+    """
+    with open(filename, 'wb') as handle:
+        response = requests.get(url, stream=False)
+
+        if response.status_code == 200 :
+            for block in response.iter_content(1024):
```

Comment:
Based on http://stackoverflow.com/questions/236861/how-do-you-determine-the-ideal-buffer-size-when-using-fileinputstream probably 4KB or 16KB should fit better.

I would parameterize this value in the function and set a default of 4096.